src/js/control-bar/control-bar.js

```diff
@@ -56,7 +55,8 @@ class ControlBar extends Component {
 ControlBar.prototype.options_ = {
   children: [
     'playToggle',
-    'volumeMenuButton',
+    'muteToggle',
```

Comment:
I am thinking that we can make `MuteToggle` and `VolumeBar` both a child of `VolumeControl`, what do you think?